test/lib/calendar_test.exs

```diff
@@ -20,4 +20,29 @@ defmodule Pairmotron.CalendarTest do
       refute Calendar.same_week?(2016, 1, ~D(2015-01-01))
     end
   end
+
+  describe "past_week?" do
+    test "same week returns false" do
+      refute Calendar.past_week?(2015, 53, ~D(2016-01-01))
+    end
+
+    test "future week returns false" do
+      refute Calendar.past_week?(2016, 10, ~D(2016-01-01))
+    end
+
+    test "past week in same year returns true" do
+      assert Calendar.past_week?(2016, 1, ~D(2016-10-01))
+    end
+
+    test "past week in past year returns true" do
+      assert Calendar.past_week?(2015, 50, ~D(2016-02-01))
+    end
+  end
+
+  describe "first_date_of_week" do
+    test "returns the date of the first day of the given week" do
+      assert Calendar.first_date_of_week(2016, 5) == ~D(2016-02-01)
+      assert Calendar.first_date_of_week(2015, 53) == ~D(2015-12-28)
+    end
```

Comment:
Looks like it is doing things mostly correctly for ISO weeks, but I did find this:

```
>> Pairmotron.Calendar.first_date_of_week(2004, 1)
** (FunctionClauseError) no function clause matching in Timex.Helpers.iso_day_to_date_tuple/2
    (timex) lib/timex/helpers.ex:6: Timex.Helpers.iso_day_to_date_tuple(2004, -2)
    (timex) lib/timex.ex:612: Timex.from_iso_triplet/1
```